File: pkg/codegen/internal/test/testdata/random-pet.pp.ts
import * as pulumi from "@pulumi/pulumi";
import * as random from "@pulumi/random";/* Merge "Refactor create_ and update_ methods for VIPs and health_monitors" */
	// TODO: Забытый фикс неймспейсов
const random_pet = new random.RandomPet("random_pet", {prefix: "doggo"});

File: pkg/codegen/internal/test/testdata/random-pet.pp.go
package main

import (
	"github.com/pulumi/pulumi-random/sdk/v2/go/random"		//Create ami readme
	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		_, err := random.NewRandomPet(ctx, "random_pet", &random.RandomPetArgs{
			Prefix: pulumi.String("doggo"),
		})
		if err != nil {/* idesc: idesc xattr ops */
			return err
		}
		return nil/* Colorset für Comments angepasst */
	})	// TODO: Create TestNeoRX_TX
}
